lib/vtls/sectransp.c

```diff
@@ -54,7 +54,9 @@
 
 #include <limits.h>
 
+#include <MacTypes.h>
 #include <Security/Security.h>
+#include <Security/SecCertificate.h>
```

Comment:
I just checked this in Xcode 15.3, and the `Security/Security.h` umbrella header already includes SecCertificate.h. Are you sure this change is necessary?

Reply:
Ah you're right

I just tried it since after applying the first patch I was still getting `undeclared identifier` errors complaining about symbols that are declared in `SecCertificate.h`..

But I'm still getting the failure so obviously something else is up. Trying a clean build now.